runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -30,53 +30,97 @@
 import net.runelite.client.config.ConfigItem;
 
 @ConfigGroup(
-	keyName = "npcindicators",
-	name = "NPC Indicators",
-	description = "Configuration for the NPC indicators plugin"
+		keyName = "npcindicators",
+		name = "NPC Indicators",
+		description = "Configuration for the NPC indicators plugin"
 )
 public interface NpcIndicatorsConfig extends Config
 {
 	@ConfigItem(
-		position = 0,
-		keyName = "npcToHighlight",
-		name = "NPCs to Highlight",
-		description = "List of NPC names to highlight"
+			position = 0,
+			keyName = "renderStyle",
+			name = "Render Style",
+			description = "Highlight setting"
+	)
+	default RenderStyle RenderStyle()
+	{
+		return RenderStyle.HULL;
+	}
+
+	@ConfigItem(
+			position = 1,
+			keyName = "npcToHighlight",
+			name = "NPCs to Highlight",
+			description = "List of NPC names to highlight"
 	)
 	default String getNpcToHighlight()
 	{
 		return "";
 	}
 
 	@ConfigItem(
-		position = 1,
-		keyName = "npcColor",
-		name = "Highlight Color",
-		description = "Color of the NPC highlight"
+			position = 2,
+			keyName = "npcColor",
+			name = "Highlight Color",
+			description = "Color of the NPC highlight"
 	)
 	default Color getNpcColor()
 	{
 		return Color.CYAN;
 	}
 
 	@ConfigItem(
-		position = 2,
-		keyName = "enableTag",
-		name = "Enable Tag Option",
-		description = "Enable the NPC tag menu option"
+			position = 3,
+			keyName = "enableTag",
+			name = "Enable Tag Option",
+			description = "Enable the NPC tag menu option"
 	)
 	default boolean isTagEnabled()
 	{
 		return false;
 	}
 
 	@ConfigItem(
-		position = 3,
-		keyName = "tagColor",
-		name = "Tag Color",
-		description = "Color of the NPC tag highlight"
+			position = 4,
+			keyName = "tagColor",
+			name = "Tag Color",
+			description = "Color of the NPC tag highlight"
 	)
 	default Color getTagColor()
 	{
 		return Color.CYAN;
 	}
-}
+
+	@ConfigItem(
+			position = 5,
+			keyName = "outlineWidth",
+			name = "Outline Width",
+			description = "Size of the outline"
+	)
+	default int outlineWidth()
```

Comment:
Is this option really needed/useful? It should just follow other defaults (like agility plugin etc) for the highlighting, no need to have the outline customizable